### PR TITLE
fix: dont "filetype off" unless did_load_filetypes

### PR DESCRIPTION
**Pull request by lervag**
This resolves the issue described in #1175.

### CODE REVIEW

File: plug.vim [hunk 1]
```diff
@@ -352,7 +352,7 @@ function! plug#end()
   endif
   let lod = { 'ft': {}, 'map': {}, 'cmd': {} }
 
-  if exists('g:did_load_filetypes')
+  if get(g:, 'did_load_filetypes', 0) > 0
```
Comment (junegunn) [inline plug.vim:355]: Let's drop `> 0` as @Shougo initially suggested.

**Review by junegunn — CHANGES_REQUESTED** [review]

File: plug.vim (hunk 1 above)
Comment (lervag) [reply plug.vim]: Done!

**Comment by junegunn** [comment]
Merged, thanks!

**Comment by lervag** [comment]
Great! Thanks for providing this excellent plugin manager! I've been a happy user since way back :D

**Pull request merged**